### PR TITLE
3521 - adds a util to check the licence info

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

follow-up of #3521

### Description
adds a utility to `runtests.sh` to check the copyright header

### Status
**Ready** (depends on #3522)

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: runtests.sh [hunk 1]
```diff
@@ -238,6 +239,7 @@ do
             doFlake8Format=true
             doPytypeFormat=true
             doMypyFormat=true
+            doCopyRight=true
         ;;
         --disttests)
```
Comment (Nic-Ma) [inline runtests.sh:244]: Could you please help add a command arg for it? Maybe `--copyright`.
Others look good to me.

Thanks.

File: runtests.sh [hunk 2]
```diff
@@ -341,6 +344,22 @@ compile_cpp
...
+    # check copyright headers
+    find "$(pwd)/monai" "$(pwd)/tests" -type f \
+      ! -wholename "*_version.py" -and -name "*.py" -or -name "*.cpp" -or -name "*.cu" -or -name "*.h" |\
+      while read -r fname; do
+        if ! grep "http://www.apache.org/licenses/LICENSE-2.0" "$fname" > /dev/null; then
+            print_error_msg "Missing the license header in file: $fname"
+            echo "Please add the licensing header to the file."
+            echo "  See also: https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#checking-the-coding-style"
+            echo ""
+            exit 1
```
Comment (Nic-Ma) [inline runtests.sh:358]: I think better to continue the check and print all the missing files instead of `exit 1`.

Thanks.

**Review by Nic-Ma — APPROVED** [review]

File: runtests.sh (hunk 1 above)
Comment (wyli) [reply runtests.sh:246]: thanks, new option added.

File: runtests.sh (hunk 2 above)
Comment (wyli) [reply runtests.sh]: yes, now it will finish the loop and print the final results

**Review by Nic-Ma — APPROVED** [review]
Thanks for the update, looks good to me.

**Pull request merged**